tutorials/debugging/holoscan_container_vscode/README.md

```diff
@@ -58,9 +58,9 @@ The `My Workspace` section contains all the files that you've copied. The `Holos
 > 💡 Tip: Open this `README.md` file in the Dev Container will help you navigate to the linked files faster.
      This file can be found inside the Dev Container under `/workspace/my/README.md`.
 
-This section will walk you through the steps to debug the [Hello World](../holoscan-sdk/examples/hello_world/README.md) application using VS Code.
+This section will walk you through the steps to debug the [Hello World](https://github.com/nvidia-holoscan/holoscan-sdk/blob/main/examples/hello_world/README.md) application using VS Code.
 
-Open the [hello_world.cpp](../holoscan-sdk/examples/hello_world/cpp/hello_world.cpp#L55) file in VS Code. (If the link doesn't work, click on **Explorer** from the sidebar and expand the **holoscan-sdk** folder. Find and open `hello_world.cpp` under the `examples/hello_world/cpp` directory.
+Open the [hello_world.cpp](/../holoscan-sdk/examples/hello_world/cpp/hello_world.cpp#L55) file in VS Code. (If the link doesn't work, click on **Explorer** from the sidebar and expand the **holoscan-sdk** folder. Find and open `hello_world.cpp` under the `examples/hello_world/cpp` directory.
```

Comment:
please double check the corresponding url on https://nvidia-holoscan.github.io/holohub after merge